stepfunctions-httpendpoint-sam/README.md

```diff
@@ -0,0 +1,87 @@
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/stepfunctions-httpendpoint-sam
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd stepfunctions-httpendpoint-sam
+    ```
+3. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy --guided
+    ```
+4. During the prompts:
+    * Enter a stack name
+    * Enter `us-east-1` or any AWS Region where you have access. 
+    * Allow SAM CLI to create IAM roles with the required permissions.
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+5. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
+
+## How it works
+
+* Start the Standard Workflow using the `start-execution` api command with a JSON input payload.
+* The Workflow directly invokes the third-party API.
+* EventBridge API Connection is used for the authentication of the third-party API. 
+
+Please refer to the architecture diagram below:
+
+![End to End Architecture](image/architecture.png)
+
+
+## Testing
+
+1. Run the following AWS CLI command to start the Step Functions workflow. Note, you must edit the {StateMachineHTTPEndpointArn} placeholder with the ARN of the deployed Step Functions workflow. This is provided in the stack outputs. Please replace {your-region} with the region selected at the time of deployment.
+
+```bash
+aws stepfunctions start-execution  --name "test" --state-machine-arn "{StateMachineHTTPEndpointArn}" --input "" --region {your-region}
```

Comment:
instead of mentioning "" in input, you may mention something like ```{stringifiedJson}``` and add an example command below. Users may make mistakes or face errors when using current command.

Reply:
Updated.

Reply:
Thanks. I meant if you would like to pass input, we could pass a JSON and could show an example. If input wasn't needed we could just remove it. However, thanks for still adding an example.